src/commons/link-replacer.ts

```diff
@@ -1,20 +1,20 @@
 const issue = ['issue'] as const;
-const report = ['condition report', 'report', 'rapport', 'meetrapport'] as const;
+const report = ['condition report', 'report', 'measurement report', 'rapport', 'meetrapport', 'conditierapport'] as const;
 const command = ['command', 'opdracht'] as const;
 
 // the regexp have been verified on https://makenowjust-labs.github.io/recheck
 const matchers = [
   [
     'issues',
-    new RegExp(`(?<key>${issue.join('|')}) #(?<hashid>[\\w\\d]{6,})`, 'gi'),
+    new RegExp(`(?<key>${issue.join('|')}) #(?<hashid>[\\w]{6,})`, 'gi'),
```

Comment:
Now that I think about this change, isn't `\w` too loose for our hashids ?
We don't have `_` in our hashids.

So it should be `[a-zA-Z\\d]` (or `[a-zA-Z0-9]` to make it simple, or even `[a-z0-9]` to be more strict) ?

Reply:
fair enough!